src/guided_fuzzing_daemon/libfuzzer.py

```diff
@@ -734,19 +716,14 @@ def libfuzzer_main(
                     numFrames=8,
                 )
                 collector.submit(crash_info, testcase)
-                print("Successfully submitted crash.", file=sys.stderr)
+                LOG.info("Successfully submitted crash.")
     finally:
-        try:
-            # Before doing anything, try to shutdown our monitors
-            for monitor in monitors:
-                if monitor is not None:
-                    monitor.terminate()
-                    monitor.join(10)
-        finally:
-            if sys.exc_info()[0] is not None:
-                # We caught an exception, print it now when all our monitors are
-                # down
-                print_exc()
+        LOG.info("Shutting down...")
```

Comment:
You code do something like:
```
if sys.exc_info()[0] is not None:
    LOG.warning("Unexpected exception!")
```

Reply:
This is the second exception we were seeing. I think the intent was to delay printing the exception until after monitors are down, but in reality the exception message was already printed so we just saw it twice.
Adding the log print would just tell us what we already know, since the exception was already printed.